docs/index.md

```diff
@@ -47,6 +47,7 @@ Besides running Zigbee2MQTT on [bare-metal Linux](getting_started/running_zigbee
 * [How to support new devices](how_tos/how_to_support_new_devices.md)
 * [How to support new Tuya compatible devices](how_tos/how_to_support_new_tuya_devices.md)
 * [How to switch to the dev branch](how_tos/how-to-switch-to-dev-branch.md)
+* [How to test your development inside the Home Assistant Addon](how_tos/how-to-test-your-development-inside-the-home-assistant-addon.md)

```

Comment:
I think: "How to develop in the Home Assistant addon" fits better here.

Reply:
Edit: wondering if it would be better to just directly link to the contributing.md instead (ofc I will add it to the contributing.md first)

Reply:
I agree with the contributing bit, as long as it exists.

Tbh this is what i wanted to do. I wanted to link to contributing.md in the ha addon, but discovered that the guide was missing, hence me showing you the merged commit that is missing.

Let me know what would you like me to do, i don't mind changing to fit the needs

Reply:
It's available now: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/blob/master/CONTRIBUTING.md

Reply:
I have now pushed a commit to resolve the above as discussed